### PR TITLE
doc: fix phrasing and whitespace

### PR DESCRIPTION
**Pull request by igrekus**
some phrasing and whitespace clean up for the "Hello world" section

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -105,32 +105,32 @@ fn main() {
...
+The return type is specified after the function name.
+In this case `main` doesn't return anything, so the return type can be omitted.
 
-As in many other languages (such as C, Go and Rust), `main` is an entry point.
+As in many other languages (such as C, Go and Rust), `main` is the entry point of your program.
 
-`println` is one of the few built-in functions. It prints the value passed to it
-to standard output.
+`println` is one of the few built-in functions.
+It prints the value passed to it to the standard output.
```
Comment (ntrel) [inline doc/docs.md:127]: I think the original is more correct, without 'the'.

**Comment by spytheman** [comment]
I agree with @ntrel . The original wording is clearer.

**Comment by ntrel** [comment]
@spytheman there are some other improvements in this pull though that could be merged, perhaps reopen this?

**Comment by igrekus** [comment]
Do I fix tihs pull request or it's better open another?

**Comment by igrekus** [comment]
@spytheman do I fix tihs pull request or it's better to open another?

**Review by ntrel — APPROVED** [review]

**Comment by spytheman** [comment]
Sorry, I was not careful enough @igrekus .

**Pull request merged**